drivers/soundwire/dmi-quirks.c

```diff
@@ -130,6 +130,14 @@ static const struct dmi_system_id adr_remap_quirk_table[] = {
 		},
 		.driver_data = (void *)intel_rooks_county,
 	},
+	{
```

Comment:
@mosomate please modify the commit title include the prefix and add a proper commit message along with your sign-off and link to the issue.

Reply:
@ranj063 I modified the commit message. I hope it fulfills the guidelines now. I'm sorry, this is my first time contributing to the linux kernel.